website/docs/r/database_secret_backend_connection.md

```diff
@@ -460,7 +460,9 @@ See the [Vault
 
 * `username` - (Optional) The username to be used in the connection (the account admin level).
 
-* `password` - (Optional) The password to be used in the connection.
+* `password` - **Deprecated** (Optional) The password to be used in the connection. Please migrate to key-pair authentication by [November 2025](https://www.snowflake.com/en/blog/blocking-single-factor-password-authentification/).
+
+* `private_key` - (Optional) The private key associated with the Snowflake user.
```

Comment:
Since this is a Write-Only field, documentation is slightly different just to call it out better. Here is the [example for the `password_wo` field](https://github.com/hashicorp/terraform-provider-vault/blob/main/website/docs/r/database_secret_backend_connection.md?plain=1#L494-L499). Since this is only added to Snowflake, we can mention that this field is only added to the Snowflake DB configuration options

Could we also add a doc line for the `_wo_version` field as done [here](https://github.com/hashicorp/terraform-provider-vault/blob/main/website/docs/r/database_secret_backend_connection.md?plain=1#L388)?